README.md

```diff
@@ -34,15 +42,84 @@ pip install -r requirements.txt
 
-To use it in earnest collect your data in an excel sheet conforming to the sample [data](data) provided or create a cvs file containing your blocker data, e.g. in the [client-data](client-data) subdirectory
+### Using an Excel Data Source
 
-1. Export your data to `./client-data/BlockersPrepared.csv`
-2. `python3 plotly-blocker-treemap.py -s -o report.html --override-field data/only-selected-fields.txt client-data/BlockersPrepared.csv`
+To use it in earnest collect your data in an excel sheet conforming to the
+sample [data](data). Then run the script
+
+```shell
+# Show properties of the table cells
+python ./plotly-blocker-treemap.py -s --info "./data/ClientBlockersForTreemap.xlsx"
+
+# Create the treemap and open it in the browser
+python ./plotly-blocker-treemap.py --outfile report.html --override-fields "./data/only-selected-fields.txt" "./data/ClientBlockersForTreemap.xlsx" 
+```
+
+If your excel sheet contains serveral tables, then use the `--tab "SheetName"`
+parameter to select the sheet for processing.
+
+### Using a CSV Data Source
+
+As an alternative, you can provide a csv file containing your blocker data. For
+the sample in the [data](data) folder, run
+
+```shell
+# Show properties of the table cells
+python .\plotly-blocker-treemap.py -s --force-csv --info "./data/BlockersPrepared.csv"
+
+# Create the treemap and open it in the browser
+python .\plotly-blocker-treemap.py --outfile report.html --force-csv --override-fields "./data/only-selected-fields.txt" "./data/BlockersPrepared.csv" 
+```
+
+## Description of Columns in the `data/ClientBlockersForTreemap.xlsx` Demo File
+
+### Duration
+
+Contains a decimal value telling how many allocated hours have passed until the blocker was solved.
+
+### Explanation
+
+Contains a short text summarizing the blocker.
+
+### Category
+
+The values in this column are categories of similar blockers, e.g. `meetings`,
+`rework`, `wait for internal person(s)`, `wait for external person(s)`,
+`capacity exhausted / priority changed`, `policy agreements violated`.
+
+The idea of this column is to discuss within the group. We want to understand
+which kind of blocker(s) we face. While discussing we learn about the ways we
+can be blocked.
+
+For categories with the largest sum of blocked time we design experiments in
+order to reduce the blockers. It is important to use the [scientific
+method](https://en.wikipedia.org/wiki/Scientific_method) for these experiments:
+create a hypothesis via induction based on observations, design the experiment,
+collect data, refine or deny the hypothesis using the data and reconsider the
+solution approach.
+
+### WorkOrIdleTime
+
+The value `work` in this column means, that the blocker required extra work in
+order to be resolved.
+
+By `idle` we tag blockers, which could only be resolved by waiting. The blockers
+did not denote extra work but for example delayed hand-overs. During the time of
+the blocker, some people were idle.
```

Comment:
A blocker indicates whether the work-item was idle. Whether the workers where idle is a question of utilization and (usually) not directly visible in the blockers.